components/dashboard/src/AppNotifications.tsx

```diff
@@ -15,7 +15,7 @@ import { User as UserProtocol } from "@gitpod/gitpod-protocol";
 import { User } from "@gitpod/public-api/lib/gitpod/v1/user_pb";
 
 const KEY_APP_DISMISSED_NOTIFICATIONS = "gitpod-app-notifications-dismissed";
-const PRIVACY_POLICY_LAST_UPDATED = "2023-10-17";
+const PRIVACY_POLICY_LAST_UPDATED = "2023-12-11";
```

Comment:
Is it worth to add an endpoint to website to get last updated date automatically?